src/siwe-client.ts

```diff
@@ -87,21 +84,19 @@ export class SiweImpl implements SiweApiClient {
       throw new Error(`Received error response on login: ${responseText}`)
     }
 
-    const headerSetCookie = (response.headers as any).raw()[
-      'set-cookie'
-    ] as Array<string>
-
-    if (headerSetCookie) {
-      await Promise.all(
-        headerSetCookie.map(async (cookie: string) => {
-          await this.cookieJar.setCookie(cookie, this.config.loginUrl)
-        }),
-      )
-    }
+    await this._extractCookies(response.headers)
 
     this._sessionExpiry = expirationTime
   }
 
+  /**
+   * Extracts cookies and stores in local variable
+   * _extractCookies is overidded in each index, per environment, thus default implementation just avoids errors
+   */
+  async _extractCookies(_headers?: Headers): Promise<void> {
```

Comment:
Since it seems like we always want to override `_extractCookies`, could it make sense to turn `SiweImpl` into an abstract class, and require `_extractCookies` as an abstract protected method that subclasses must implement? I guess the browser client doesn't set `_extractCookies`, but we could just use the empty object implementation there.